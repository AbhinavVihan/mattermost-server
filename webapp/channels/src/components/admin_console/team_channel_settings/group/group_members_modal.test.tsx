// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {TestHelper} from 'utils/test_helper';

import GroupMembersModal from './group_members_modal';
import {Provider} from 'react-redux';
import store from 'stores/redux_store';
import {renderWithIntl} from 'tests/react_testing_utils';

describe('admin_console/team_channel_settings/group/GroupList', () => {
    test('should match snapshot while visible', () => {
        const group = TestHelper.getGroupMock({});

        const wrapper = renderWithIntl(
            <Provider store={store}>
                <GroupMembersModal
                    group={group}
                    onExited={jest.fn()}
                />
            </Provider>,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
