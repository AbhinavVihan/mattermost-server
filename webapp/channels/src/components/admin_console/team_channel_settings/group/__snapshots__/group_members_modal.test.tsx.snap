// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`admin_console/team_channel_settings/group/GroupList should match snapshot while visible 1`] = `
<div
  aria-hidden="true"
/>
`;
